123_state_tasks.R

```diff
@@ -60,14 +61,20 @@ do_state_tasks <- function(oldest_active_sites, ...) {
     include = c('remake.yml'),
     packages = c("tidyverse", "dataRetrieval", "lubridate"),
     sources = c(...),
-    tickquote_combinee_objects = FALSE,
-    finalize_funs = c())
+    final_targets = c('obs_tallies', '3_visualize/out/timeseries_plots.yml'),
+    finalize_funs = c('combine_obs_tallies', 'summarize_timeseries_plots'),
+    as_promises = TRUE,
+    tickquote_combinee_objects = TRUE)
 
   # Build the tasks
-  scmake('123_state_tasks', remake_file='123_state_tasks.yml')
+  obs_tallies <- scmake('obs_tallies_promise', remake_file='123_state_tasks.yml')
+  scmake('timeseries_plots.yml_promise', remake_file='123_state_tasks.yml')
+  timeseries_plots_info <- yaml::yaml.load_file('3_visualize/out/timeseries_plots.yml') %>%
+    tibble::enframe(name = 'filename', value = 'hash') %>%
+    mutate(hash = purrr::map_chr(hash, `[[`, 1))
 
   # Return nothing to the parent remake file
```

Comment:
If I remember right, I added a note to encourage folks to edit this comment when they changed the `return()` call, with the idea of fostering that good habit of checking and updating comments whenever you update associated code. The actual comment is obviously not important here because this is just an exercise...but consider this note your bonus encouragement to keep on keeping comments fresh in real projects, which I think you're generally pretty good at.

Reply:
Ah thanks for pointing that out